source/blog/2018-04-01-the-emberjs-times-issue-43.md

```diff
@@ -43,13 +43,37 @@ and give your thoughts and suggestions in the discussion.
   <p>
     With Ember 3.1 you now get ES5 getters for computed properties since our target browsers support ES5 getters. The motivation is to improve developer ergonomics and interoperability with other tooling and libraries. For the property to recompute you still need to use this.set() or Ember.set().</p>
   <p>
-    Yet, this doesn’t mean that this.get() is now deprecated. There’s actually still a lot of cases where you should use this.get() instead of the ES5 getter. 
+    Yet, this doesn’t mean that this.get() is now deprecated. There’s actually still a lot of cases where you should use this.get() instead of the ES5 getter.
     You can read all about it in the <a href="https://www.emberjs.com/blog/2018/04/13/ember-3-1-released.html#toc_es5-getters-for-computed-properties-1-of-3">release post</a>.
   </p>
 </div>
 
 <a class="ember-button ember-button--centered href="/blog/2018/04/13/ember-3-1-released.html">Read more</a>
 
+## Say More
```

Comment:
I think it would be cool to add the link to the RFC in the header title, too, what do you think?

```
## [Say More](https://github.com/emberjs/rfcs/pull/327] 
```